framework/wazuh/syscheck.py

```diff
@@ -32,20 +33,22 @@ def run(agent_list=None):
     result = AffectedItemsWazuhResult(all_msg='Syscheck scan was restarted on returned agents',
                                       some_msg='Syscheck scan was not restarted on some agents',
                                       none_msg='No syscheck scan was restarted')
+
+    wq = WazuhQueue(common.ARQUEUE)
     for agent_id in agent_list:
         try:
-            agent_info = Agent(agent_id).get_basic_information()
-            agent_status = agent_info.get('status', 'N/A')
-            if agent_status.lower() != 'active':
-                result.add_failed_item(
-                    id_=agent_id, error=WazuhError(1601, extra_message='Status - {}'.format(agent_status)))
-            else:
-                wq = WazuhQueue(common.ARQUEUE)
+            agent = Agent(agent_id)
+            agent.load_info_from_db()
+            agent_status = agent.status
```

Comment:
What happens if the agent has no status? (000)

agent.status if agent.status else 'N/A'

Reply:
Done!